src/extension.ts

```diff
@@ -188,6 +189,7 @@ export async function activate(context: vscode.ExtensionContext) {
         )
 
         await activateCloudFormationTemplateRegistry(context)
+        await activateBuildSpecTemplateRegistry(context)
```

Comment:
Even if https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html#build-spec-ref-syntax spelled it "BuildSpec" I would suggest that the programmatic symbols in our codebase do not need the entropy. We just need a nonce. Punctuation  is more meaningful if it is used sparingly.

```suggestion
        await activateBuildspecTemplateRegistry(context)
```